staging/src/k8s.io/client-go/discovery/cached/memcache.go

```diff
@@ -111,41 +129,46 @@ func (d *memCacheClient) Fresh() bool {
 	return d.cacheValid
 }
 
-// Invalidate refreshes the cache, blocking calls until the cache has been
-// refreshed. It would be trivial to make a version that does this in the
-// background while continuing to respond to requests if needed.
+// Invalidate enforces that no cached data is used in the future that is older than the current time.
```

Comment:
I know you copied it from the interface, but it's nonsensical. Can we rewrite it everywhere to "Invalidate enforces that no cached data that is older than the current time is used"?

Reply:
Done